scripts/devtools/prepare-release.js

```diff
@@ -99,28 +99,45 @@ async function getCommitLog(sha) {
   let shortLog = '';
   let formattedLog = '';
 
+  const hasGh = await hasGithubCLI();
   const rawLog = await execRead(`
     git log --topo-order --pretty=format:'%s' ${sha}...HEAD -- packages/react-devtools*
   `);
-  rawLog.split('\n').forEach(line => {
-    line = line.replace('[DevTools] ', '');
-
+  for (let line of rawLog.split('\n')) {
+    line = line.replace(/^\[[dD]ev[tT]ools\]? /, '');
```

Comment:
With the `i` flag for case insensitive this simplifies a bit. I also think you have the `?` not where intended (optional `]`, but you probably wanted to strip optional spaces. ` *` matches any number of spaces.
```suggestion
    line = line.replace(/^\[devtools\] */i, '');
```